crates/re_viewer_context/src/store_hub.rs

```diff
@@ -1,15 +1,14 @@
 use ahash::{HashMap, HashMapExt};
```

Comment:
I moved around some functions in this file. You can review commit-by-commit if you don't want that noise. Sorry.